### PR TITLE
Fix namespace

### LINKED ISSUE #139: Folder and namespace structure
The namespace and folder structure is not regulated at the moment, and before messing up all the things, it is important to decide a way on how to organize existing code and give guidelines for future code.

### PR DESCRIPTION
**Pull request by rasolca**
Finished removal of dlaf_test namespace.

Closes #139.

### CODE REVIEW

**Comment by rasolca** [comment]
bors try

**Review by albestro — APPROVED** [review]
LGTM.

Just one question: do you want to keep the folder structure `test/include/dlaf_test/` as it is?

**Comment by rasolca** [comment]
Yes I will keep `test/include/dlaf_test/` such that `#include"dlaf_test/..."` immediately shows that we are including a test header.

File: test/include/dlaf_test/matrix/util_generic_blas.h [hunk 1]
```diff
@@ -23,7 +23,7 @@ namespace dlaf {
 namespace matrix {
 namespace test {
 
-using namespace dlaf_test;
+using namespace dlaf::test;
```
Comment (teonnik) [inline test/include/dlaf_test/matrix/util_generic_blas.h:26]: `using namespace` in a header is somewhat surprising (dagerous?) : https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers

File: test/include/dlaf_test/matrix/util_generic_lapack.h [hunk 2]
```diff
@@ -23,7 +23,7 @@ namespace dlaf {
 namespace matrix {
 namespace test {
 
-using namespace dlaf_test;
+using namespace dlaf::test;
```
Comment (teonnik) [inline test/include/dlaf_test/matrix/util_generic_lapack.h:26]: Here too.

File: test/include/dlaf_test/matrix/util_matrix_blas.h [hunk 3]
```diff
@@ -21,7 +21,7 @@
 namespace dlaf {
 namespace matrix {
 namespace test {
-using namespace dlaf_test;
+using namespace dlaf::test;
```
Comment (teonnik) [inline test/include/dlaf_test/matrix/util_matrix_blas.h:24]: Here.

File: test/include/dlaf_test/matrix/util_tile_blas.h [hunk 4]
```diff
@@ -21,7 +21,7 @@
 namespace dlaf {
 namespace matrix {
 namespace test {
-using namespace dlaf_test;
+using namespace dlaf::test;
```
Comment (teonnik) [inline test/include/dlaf_test/matrix/util_tile_blas.h:24]: Here.

**Review by teonnik — CHANGES_REQUESTED** [review]
There may be some places I missed.

File: test/include/dlaf_test/matrix/util_generic_blas.h (hunk 1 above)
Comment (rasolca) [reply test/include/dlaf_test/matrix/util_generic_blas.h]: Correct!

**Review by teonnik — APPROVED** [review]

**Comment by msimberg** [comment]
Unrelated to this PR in particular, but have you tried `bors merge` or `bors r+`? That should do the merge automatically if tests on a merge with master passes. That should be safer, no?

**Comment by rasolca** [comment]
The use of these commands has been considered, but unfortunately they don't give the opportunity to provide a clean squash-merge commit custom message.
FYI: the CI build system is sometimes unstable and might fail without any reason. If you see the following error just restart the failed CI task.
```
ERROR: Job failed (system failure): unable to upgrade connection: Authorization error (user=kubelet-client, verb=create, resource=nodes, subresource=proxy)
```

**Pull request merged**